doc/src/modules/physics/mechanics/system.rst

```diff
@@ -0,0 +1,95 @@
+.. _system:
+
+===========================
+System in Physics/Mechanics
+===========================
+
+:mod:`sympy.physics.mechanics` provides a :class:`~.System` class. In a
+:class:`~.System` instance you store information of the model, e.g. bodies,
+joints, constraints, etc. With all relations of the :class:`~.System` being
+defined it can form the equations of motion using a backend of your choosing.
+In general :class:`~.System` has been designed to be compatible with third party
+libraries, like `PyDy`.
```

Comment:
- Might be useful to specify that "backend" in this context means `KanesMethod` or `LangrangesMethod`?
- Do we need to link to PyDy?
```suggestion
:mod:`sympy.physics.mechanics` provides a :class:`~.System` class. In a
:class:`~.System` instance you store information about the model, e.g. bodies,
joints, constraints, etc. With all relations of the :class:`~.System` being
defined it can form the equations of motion using a backend of your choosing.
In general :class:`~.System` has been designed to be compatible with third party
libraries, like `PyDy`.
```

Reply:
No I'll remove `PyDy`, as we don't the link yet anyway.